nixos/modules/programs/zsh/spaceship-prompt.nix

```diff
@@ -0,0 +1,28 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.zsh.spaceship-prompt;
+in {
+  options = {
+    programs.zsh.spaceship-prompt = {
+      enable = mkEnableOption "spaceship-prompt";
+      package = mkOption {
```

Comment:
There is only one option with no variants. This option seems pointless.

Reply:
What do you mean? You could easily supply a package that is not e.g. in nixpkgs if you are making a derivation from master. You could always use an overlay for this as well, but its pretty standard practice to allow the overriding of the package in modules like this.

Reply:
> You could always use an overlay for this

Yes, that was my point...